packages/angular/build/src/tools/vite/middlewares/assets-middleware.ts

```diff
@@ -7,6 +7,9 @@
  */
 
 import { lookup as lookupMimeType } from 'mrmime';
+import { createHash } from 'node:crypto';
```

Comment:
This seems unrelated to the version bump / upstream fix for the source map issue. At least https://github.com/angular/angular-cli/pull/30770 appears to pass CI without further changes. Can you add some context to the PR description?

Reply:
This is because it was needed to bump to vite 7.0.5